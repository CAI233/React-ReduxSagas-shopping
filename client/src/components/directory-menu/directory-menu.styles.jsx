import styled from 'styled-components';

export const DirectoryMenuStyledContainer = styled.div`
    width: 100%;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
`;

DirectoryMenuStyledContainer.displayName = "DirectoryMenuStyledContainer";