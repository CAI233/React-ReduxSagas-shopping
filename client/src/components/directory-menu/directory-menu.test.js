import React from 'react';
import { shallow } from 'enzyme';
import { DirectoryMenu } from './directory-menu.component';

describe(" 测试主页目录显示 ",()=>{
    let wrapper,mockProps;
    beforeEach(()=>{
        mockProps = {
            sections: [
                {
                    id: 1,
                },
                {
                    id: 2,
                }
            ]
        };
        wrapper = shallow( <DirectoryMenu { ...mockProps } /> );
    });
    it("DirectoryMenu Snapshot",()=>{
        expect(wrapper).toMatchSnapshot();
    });

});