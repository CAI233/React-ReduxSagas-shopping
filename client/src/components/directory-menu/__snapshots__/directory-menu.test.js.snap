// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 测试主页目录显示  DirectoryMenu Snapshot 1`] = `ShallowWrapper {}`;
